### PR TITLE
Fixed IE9 failing test cases

### PR DESCRIPTION
**Pull request by obaidott**
Added wrapper to handle console.log statements to make sure tests pass on IE9

### CODE REVIEW

File: test/basics.html [hunk 1]
```diff
@@ -23,15 +34,22 @@
 <script src='../node_modules/steal/steal.js' main='@empty'></script>
 <script src='../../../node_modules/steal/steal.js' main='@empty'></script>
 <script>
+  var fixture = document.getElementById('steal-done');
```
Comment (phillipskevin) [inline test/basics.html]: Are these changes needed?

**Comment by matthewp** [comment]
We aren't going to be supporting IE9 in the next release, so closing this (thanks anyways!)
